mucommander-core/build.gradle

```diff
@@ -52,7 +82,7 @@ jar {
         'Implementation-Version': revision.substring(0, 7),
         'Build-Date': new Date().format('yyyyMMdd'),
         'Build-Url': "https://www.mucommander.com/version/nightly.xml",
-        'Import-Package': 'org.violetlib.aqua;resolution:=dynamic,com.formdev.flatlaf;resolution:=dynamic,com.apple.*;resolution:=dynamic,sun.security.action;resolution:=dynamic,*',
+        'Import-Package': 'org.violetlib.aqua;resolution:=dynamic,com.formdev.flatlaf;resolution:=dynamic,kotlin.internal.*;resolution:=dynamic,javax.jms;resolution:=dynamic,com.ibm.uvm.tools;resolution:=dynamic,sun.swing;resolution:=dynamic,org.apache.log4j;resolution:=dynamic,java.awt.*;resolution:=dynamic,sun.awt.*;resolution:=dynamic,kotlin.reflect.jvm.internal;resolution:=dynamic,com.sun.*;resolution:=dynamic,apple.awt.*;resolution:=dynamic,com.apple.*;resolution:=dynamic,sun.security.action;resolution:=dynamic,*',
```

Comment:
I tried to run the app with this change and mucommander-core failed with
```
org.osgi.framework.BundleException: Unable to resolve mucommander-core [102](R 102.0): missing requirement [mucommander-core [102](R 102.0)] osgi.ee; (&(osgi.ee=JavaSE)(version=14)) Unresolved requirements: [[mucommander-core [102](R 102.0)] osgi.ee; (&(osgi.ee=JavaSE)(version=14))]
```

I don't see yet where we added that dependency of Java 14 (I use Java 17), do you know?

Reply:
I don't have a clue. What bothers me also (like I said in previous comments) is Windows/Linux and whether those imports will be fine for other OSes.

Reply:
those 'dynamic' ones shouldn't harm